pkg/planner/cardinality/row_count_column.go

```diff
@@ -312,7 +313,17 @@ func GetColumnRowCount(sctx context.PlanContext, c *statistics.Column, ranges []
 		}
 		rowCount += cnt
 	}
-	rowCount = mathutil.Clamp(rowCount, 0, float64(realtimeRowCount))
+	allowZeroEst := fixcontrol.GetBoolWithDefault(
+		sctx.GetSessionVars().GetOptimizerFixControlMap(),
+		fixcontrol.Fix47400,
+		false,
+	)
+	if allowZeroEst {
```

Comment:
Does this mean we will always enable this optimization? So this fix control is for the users who want to use the old behavior, am I understand it correctly?

Reply:
I found it in the document PR.

Reply:
Yes.